region/brinstar/green/Green Brinstar Beetom Room.json

```diff
@@ -140,6 +140,36 @@
         "FIXME: Using a Missile, Super, or Power Bomb at the start could be added to the logic to get the first Power Bombs to use for further farming, but this needs a way to express that we're not in health-bomb energy range."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "Crystal Flash",
+      "requires": [
+        {"or": [
+          {"obstaclesCleared": ["A"]},
+          {"enemyDamage": {
+            "enemy": "Beetom",
+            "type": "contact",
+            "hits": 1
+          }}
+        ]},
+        "h_canCrystalFlash"
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "10 Power Bomb Crystal Flash",
+      "requires": [
+        {"or": [
+          {"obstaclesCleared": ["A"]},
```

Comment:
Parts of clearing A can work, like using Ice.  But you need the Beetoms to drop PBs.

`can10PowerBombCrystalFlash` is important to not be in health bomb so we'll need a way to check for that.

Reply:
If you're in health bomb range, it should be possible to farm a few times to get above 50 energy. You'll have 10 PBs which should be more than enough. I'm assuming that you can reset the room here. I can add that as a requirement.